### PR TITLE
floodplains steward requested changes

### PR DESCRIPTION
**Pull request by rkelson**
misc steward requested changes

### CODE REVIEW

File: data/water/flood-areas/index.html [hunk 1]
```diff
@@ -21,10 +21,10 @@
   <figcaption class="caption__text">Floodplains</figcaption>
 </figure>
 {% include abstract.html
-name="Floodplains"
+name="Special Flood Hazard Areas"
 stewards="Utah Division of Emergency Management and FEMA"
-abstract="The Floodplains data contains the Digital Flood Insurance Rate Map (DFIRM) Database depicting flood risk
-information and supporting data used to develop the risk data. The primary risk classifications used are the
+abstract="The Floodplains dataset refers the Digital Flood Insurance Rate Map (DFIRM) Database depicting flood risk
```
Comment (steveoh) [inline data/water/flood-areas/index.html:26]:
```suggestion
abstract="The Floodplains dataset refers to the Digital Flood Insurance Rate Map (DFIRM) Database depicting flood risk
```

File: data/water/flood-areas/index.html [hunk 2]
```diff
@@ -33,45 +33,61 @@
...
+        land facilities located or to be located in identified areas having special flood hazards," Section 2 (b) (4) of 
+        the Flood Disaster Protection Act of 1973. Special Flood Hazard Areas (SFHAs) identified on the map are the risk 
+        zones shown on the FIRMs and are the basis for the establishment of premium rates for flood coverage offered 
+        through the NFIP.</p>
+
+    <p>The Digital Flood Insurance rate Map (DFIRM) Database presents the flood risk information depicted on the FIRM in 
+        a digital format suitable for use in electronic mapping applications. The DFIRM database is a subset of the 
+        Digital Flood Insurance Study (FIS) database that serves to archive the information collected during the FIS.</p>
+
+    <p>Floodp Zone information can be found at the <a href="http://msc.fema.gov/portal"> FEMA Map Service Center
```
Comment (steveoh) [inline data/water/flood-areas/index.html]: Is Floodp a spelling error?
```suggestion
    <p>Floodp Zone information can be found at the <a href="https://msc.fema.gov/portal"> FEMA Map Service Center
```

File: data/water/flood-areas/index.html [hunk 3]
```diff
@@ -33,45 +33,61 @@
...
+        zones shown on the FIRMs and are the basis for the establishment of premium rates for flood coverage offered 
+        through the NFIP.</p>
+
+    <p>The Digital Flood Insurance rate Map (DFIRM) Database presents the flood risk information depicted on the FIRM in 
+        a digital format suitable for use in electronic mapping applications. The DFIRM database is a subset of the 
+        Digital Flood Insurance Study (FIS) database that serves to archive the information collected during the FIS.</p>
+
+    <p>Flood Zone information can be found at the <a href="http://msc.fema.gov/portal"> FEMA Map Service Center
         (MSC)</a> and in the <a
         href="http://fema.maps.arcgis.com/home/webmap/viewer.html?webmap=cbe088e7c8704464aa0fc34eb99e7f30">FEMA National
```
Comment (steveoh) [inline data/water/flood-areas/index.html:68]:
```suggestion
        href="https://fema.maps.arcgis.com/home/webmap/viewer.html?webmap=cbe088e7c8704464aa0fc34eb99e7f30">FEMA National
```

**Review by steveoh — APPROVED** [review]

File: data/water/flood-areas/index.html [hunk 4]
```diff
@@ -33,45 +33,61 @@
...
+
+    <p>Flood Zone information can be found at the <a href="https://msc.fema.gov/portal"> FEMA Map Service Center
         (MSC)</a> and in the <a
-        href="http://fema.maps.arcgis.com/home/webmap/viewer.html?webmap=cbe088e7c8704464aa0fc34eb99e7f30">FEMA National
+        href="https://fema.maps.arcgis.com/home/webmap/viewer.html?webmap=cbe088e7c8704464aa0fc34eb99e7f30">FEMA National
         Flood Hazard Layer.</a></p>
+
     <p>One thing to keep in mind with the zone designations is the difference between "Shaded X" and "Unshaded X". In
       the link above, Shaded X represents the 0.2% annual chance floodplain (the 500-year flood). In almost all of the
       DFIRMs, the ZONE field will read: 0.2 PCT ANNUAL CHANCE FLOODPLAIN or something like that, and in some DFIRMs it
```
[the remark this answers is not in the document]
Comment (rkelson) [reply data/water/flood-areas/index.html:73]: I am pasting in what they provided but I am not sure if there should be a <p> before that or not.

[the remark this answers is not in the document]
Comment (rkelson) [reply data/water/flood-areas/index.html:73]: I am going to remove the paragraph, get this merged, and let them know to have a look

[the remark this answers is not in the document]
Comment (rkelson) [reply data/water/flood-areas/index.html:73]: on second look that is how it is on the current page so I'll leave the paragraph
I think this is ready to merge

**Pull request merged**